### PR TITLE
Added existence test cases for policy and remote execution feature

### PR DESCRIPTION
**Pull request by devendra104**
We have added policy and remote execution feature's related test case in the existence test case's CLI module. 
The purpose of these test cases to do the comparison between pre and post upgrade's CLI content and to check if some unexpected changes happen.

Test Results:

```
 pytest -s upgrade_tests/test_existance_relations/cli/test_policy.py 
=============================================================================================== test session starts ===============================================================================================
                                                                                                                                                                                         upgrade_tests/test_existance_relations/cli/test_policy.py ..........
====================================================================================== 10 passed, 2 warnings in 0.26 seconds ======================================================================================
```

```
pytest -s upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py 
=============================================================================================== test session starts ===============================================================================================
                                                                                                                                                                                           upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py ........................................................

====================================================================================== 56 passed, 2 warnings in 0.33 seconds ======================================================================================

```

### CODE REVIEW

File: upgrade_tests/test_existance_relations/cli/test_policy.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+:CaseLevel: System
+
+:CaseComponent: Compliance Policy
+
+:TestType: nonfunctional
+
+:CaseImportance: High
+
+:SubType1: installability
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_policy.py:16]: Sure about installability ?

File: upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+:CaseLevel: System
+
+:CaseComponent: Remote Execution Feature
+
+:TestType: nonfunctional
+
+:CaseImportance: High
+
+:SubType1: installability
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py:16]: same here.

File: upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+:Upstream: No
+"""
+import pytest
+from upgrade_tests.helpers.common import existence
+from upgrade_tests.helpers.existence import compare_postupgrade, pytest_ids
+
+
+# Required Data
+component = 'remote-execution-feature'
+remote_execution_feature_name = compare_postupgrade(component, 'id')
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py:27]:
```suggestion
rex_feature_name = compare_postupgrade(component, 'id')
```

Comment (jyejare) [reply upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py:27]: same for all.

Comment (jyejare) [reply upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py:27]: Name -> id and id -> name issue again here.

File: upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+:Upstream: No
+"""
+import pytest
+from upgrade_tests.helpers.common import existence
+from upgrade_tests.helpers.existence import compare_postupgrade, pytest_ids
+
+
+# Required Data
+component = 'remote-execution-feature'
```
Comment (jyejare) [inline upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py:26]: Why `feature` keyword is coming there?

Comment (devendra104) [reply upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py:26]: This is the hammer component name.

**Comment by devendra104** [comment]
Test Results:

**`Test_Policy`**

```
# pytest -s upgrade_tests/test_existance_relations/cli/test_policy.py 
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.6.6, pytest-5.4.3, py-1.8.1, pluggy-0.13.1                                                                                                                                                                                       

upgrade_tests/test_existance_relations/cli/test_policy.py ..........

========================================================================================= 10 passed, 2 warnings in 0.30s ==========================================================================================

```
**Test Remote Execution**

```
pytest -s upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py 
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.6.6, pytest-5.4.3, py-1.8.1, pluggy-0.13.1                                                                                                                                                                                   

upgrade_tests/test_existance_relations/cli/test_remote_execution_feature.py ........................................................

========================================================================================= 56 passed, 2 warnings in 0.35s ==========================================================================================
```

**Comment by devendra104** [comment]
@jyejare kindly merge this PR.

**Review by jyejare — APPROVED** [review]
ACK again, We need to fix all the parmetrized tests by adding `parametrized: yes` token. But we can consider that as separate effort.

**Pull request merged**